server/src/main/java/org/apache/uniffle/server/ShuffleServerGrpcService.java

```diff
@@ -229,25 +232,24 @@ public void sendShuffleData(SendShuffleDataRequest req,
       }
       final long start = System.currentTimeMillis();
       List<ShufflePartitionedData> shufflePartitionedData = toPartitionedData(req);
+      int alreadyReleasedSize = 0;
```

Comment:
`int` is enough now, but i think `long` is more safe in the future. But `int` is ok for me too.